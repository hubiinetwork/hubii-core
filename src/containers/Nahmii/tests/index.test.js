// import React from 'react';
// import { shallow } from 'enzyme';

// import { Nahmii } from '../index';

describe('<Nahmii />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
