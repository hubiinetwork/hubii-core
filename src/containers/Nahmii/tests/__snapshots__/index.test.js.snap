// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nahmii /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nahmii
    dispatch={[Function]}
    history={
        Object {
            "location": Object {
              "pathname": "some-path",
            },
          }
    }
    intl={
        Object {
            "formatMessage": [Function],
          }
    }
    match={
        Object {
            "url": "some-url",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <TopHeader>
          <Heading>
                    my_nahmii
          </Heading>
</TopHeader>,
        <Tabs
          activeKey="some-path"
          animated={false}
          onChange={[Function]}
>
          <TabPane
                    placeholder={null}
                    tab={
                              <span>
                                        <Icon
                                                  type="solution"
                                        />
                                        airdriip_registration
                              </span>
                    }
          />
</Tabs>,
        <Route
          component={[Function]}
          path="some-url/airdriip-registration"
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Heading>
            my_nahmii
</Heading>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "my_nahmii",
          },
          "ref": null,
          "rendered": "my_nahmii",
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeKey": "some-path",
          "animated": false,
          "children": <TabPane
            placeholder={null}
            tab={
                        <span>
                                    <Icon
                                                type="solution"
                                    />
                                    airdriip_registration
                        </span>
            }
/>,
          "onChange": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": "some-url/airdriip-registration",
          "nodeType": "class",
          "props": Object {
            "placeholder": null,
            "tab": <span>
              <Icon
                            type="solution"
              />
              airdriip_registration
</span>,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "path": "some-url/airdriip-registration",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <TopHeader>
            <Heading>
                        my_nahmii
            </Heading>
</TopHeader>,
          <Tabs
            activeKey="some-path"
            animated={false}
            onChange={[Function]}
>
            <TabPane
                        placeholder={null}
                        tab={
                                    <span>
                                                <Icon
                                                            type="solution"
                                                />
                                                airdriip_registration
                                    </span>
                        }
            />
</Tabs>,
          <Route
            component={[Function]}
            path="some-url/airdriip-registration"
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Heading>
              my_nahmii
</Heading>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "my_nahmii",
            },
            "ref": null,
            "rendered": "my_nahmii",
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeKey": "some-path",
            "animated": false,
            "children": <TabPane
              placeholder={null}
              tab={
                            <span>
                                          <Icon
                                                        type="solution"
                                          />
                                          airdriip_registration
                            </span>
              }
/>,
            "onChange": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": "some-url/airdriip-registration",
            "nodeType": "class",
            "props": Object {
              "placeholder": null,
              "tab": <span>
                <Icon
                                type="solution"
                />
                airdriip_registration
</span>,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "path": "some-url/airdriip-registration",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
